src/AutoMapper/Configuration/Internal/PrimitiveHelper.cs

```diff
@@ -25,6 +25,9 @@ public static MethodInfo GetInheritedMethod(Type type, string name)
         public static MemberInfo GetFieldOrProperty(Type type, string name) 
             => type.GetInheritedMember(name) ?? throw new ArgumentOutOfRangeException(nameof(name), $"Cannot find member {name} of type {type}.");
 
+        public static bool HasFieldOrProperty(Type type, string name)
```

Comment:
@lbargaoanu where would be the test class?
a new class in the `AutoMapper.UnitTests.Bug` namesapce 
or in the `AutoMapper.UnitTest.ReverseMapIgnoreAttributeTests` class

and I want to name it `Sould_not_throw_exception_when_reverse_property_does_not_exist`, OK?